samigo/samigo-app/src/java/org/sakaiproject/tool/assessment/ui/bean/evaluation/TotalScoresBean.java

```diff
@@ -220,6 +232,53 @@ protected void init() {
 		
 		agents = newAgents;	
 	}
+
+  public boolean isOneSelectionType() {
+    if (this.getPublishedAssessment() != null) {
+      for (Object itemObject : this.getPublishedAssessment().getDefaultSection().getItemArray()) {
+        PublishedItemData item = (PublishedItemData) itemObject;
+        if (!(item.getTypeId().equals(TypeIfc.MULTIPLE_CHOICE)) && !(item.getTypeId().equals(TypeIfc.MULTIPLE_CORRECT_SINGLE_SELECTION)) && 
+					  !(item.getTypeId().equals(TypeIfc.TRUE_FALSE))) 
+				{
+          return false;
+        }
+      }
+      return true;
+    }
+    return false;
+  }
+
+  public Map getResults() {
```

Comment:
Could you add comments to these method? What it does etc..

Reply:
getResults is a very generic name, I'd like a more specific name related to the feature.